src/main/java/com/band/api/resolvers/Mutation.java

```diff
@@ -0,0 +1,35 @@
+package com.band.api.resolvers;
+
+import com.band.api.domain.User;
+import com.band.api.repository.UserRepository;
+import com.coxautodev.graphql.tools.GraphQLMutationResolver;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Component
+public class Mutation implements GraphQLMutationResolver {
+
+    @Autowired
```

Comment:
Update dependency injection to follow pattern in HelloWorldController (constructor Autowired instead of field).